src/storage/index/IndexPolicyMaker.cpp

```diff
@@ -78,6 +78,12 @@ bool IndexPolicyMaker::buildPolicy() {
         if (exist == scanItems_.end()) {
             break;
         }
+        // Stop build policy when last operator is range scan,
+        // And other fields will use expression filtering.
+        auto it = scanItems_.rbegin()->second;
```

Comment:
Is `rbegin` wrong? We can't make sure that the last index column is the `rbegin` of `scanItems`.

Reply:
> Is `rbegin` correct? We can't make sure that the last index column is the `rbegin` of `scanItems`.

I think this is correct. we just have to make sure it's the last one in scanItems_, not index fields. WDYT?

Reply:
But the scanItems_ is a map<index_column_name, ScanBound>? The `rbegin` is the last element of a map, not the column name in index order?

Reply:
> But the scanItems_ is a map<index_column_name, ScanBound>? The `rbegin` is the last element of a map, not the column name in index order?

you are right. my bad.